pdataviewer/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { TestBed } from '@angular/core/testing';
```

Comment:
Seems odd to me to have test classes in the src/app folder, is this normal?

Reply:
Not sure of that

Reply:
Just checked it again and it should be in the src/app